code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -258,14 +258,31 @@
 /mob/living/simple_animal/hostile/guardian/punch
 	melee_damage_lower = 20
 	melee_damage_upper = 20
+	armour_penetration = 75
 	damage_transfer = 0.5
-	playstyle_string = "As a standard type you have no special abilities, but have a high damage resistance and a powerful attack capable of smashing through walls."
+	playstyle_string = "As a standard type you have the ability to stop time for everyone but you and your summoner, but have a high damage resistance and a powerful attack capable of smashing through walls."
 	environment_smash = 2
 	magic_fluff_string = "..And draw the Assistant, faceless and generic, but never to be underestimated."
 	tech_fluff_string = "Boot sequence complete. Standard combat modules loaded. Holoparasite swarm online."
 	bio_fluff_string = "Your scarab swarm stirs to life, ready to tear apart your enemies."
 	var/battlecry = "AT"
 
+/mob/living/simple_animal/hostile/guardian/punch/New()
+	..()
+	var/obj/effect/proc_holder/spell/aoe_turf/conjure/timestop/holoparasite/holoparasite = null
```

Comment:
From what I've seen we usually name variables like this "HP" or something, instead of just "type" as it's very confusing right now. Especially below when you go "`holoparasite.clothes_req..`"